src/tebako-dfs.cpp

```diff
@@ -1,21 +1,30 @@
 /**
- * \author     Marcus Holland-Moritz (github@mhxnet.de)
- * \copyright  Copyright (c) Marcus Holland-Moritz
  *
- * This file is part of dwarfs.
+ * Copyright (c) 2021, [Ribose Inc](https://www.ribose.com).
```

Comment:
@maxirmx if we retain any code from an external project we need to retain the original licensing. I would suggest a rewrite of the remaining lines from the original source so that we are compliant with the license removal. Thanks.

Reply:
@ronaldtse 
The plan was (is) to rewite all remaining lines since they are too generic.

However, I just noticed that dwarFS is duistributed under GNU 3.0, not LGPL. 
Strictly speaking it means that tebako has to be GNU as well even if it is linked as a library.

Reply:
We have an agreement with the author of DwarFS that it will be re-licensed in MPL, so this is not a problem.

Reply:
It will be adressed by #10